src/mono/wasm/debugger/BrowserDebugProxy/DevToolsHelper.cs

```diff
@@ -128,39 +128,40 @@ public struct Result
         public bool IsOk => Value != null;
         public bool IsErr => Error != null;
 
-        private Result(JObject result, JObject error)
+        private Result(JObject resultOrError, bool isError, JObject error = null)
```

Comment:
Actually I leaved it only because of the FromJson function:
```
        public static Result FromJson(JObject obj)
        {
            //Log ("protocol", $"from result: {obj}");
            return new Result(obj["result"] as JObject, false, obj["error"] as JObject);
        }
```
that I did not want to change but now I did it and the version with 2 arguments works equally good. Thanks!